plugins/updater/src/config.rs

```diff
@@ -81,7 +85,8 @@ pub struct Config {
     /// Signature public key.
     pub pubkey: String,
     /// The Windows configuration for the updater.
-    pub windows: Option<WindowsConfig>,
+    #[serde(default)]
+    pub windows: WindowsConfig,
```

Comment:
I would rather keep this as `None` and find a different strategy and just apply a default installation mode if it was `None`

Reply:
Ok, I just thought this is cleaner for the consumers so that they can handle 1 less `Option`

Reply:
I prefer to write `None` that `WindowsConfig::default` when using the `Config` struct from rust.